tests/src/e2e.spec.ts

```diff
@@ -54,34 +98,92 @@ test.skip('Check welcome page is displayed and that we are redirected to the mai
          * The main window is created hidden, and is shown only when it is ready.
          * See {@link ../packages/main/src/mainWindow.ts} function
          */
+
+        mainWindow.webContents.closeDevTools();
```

Comment:
Again, here, why are you changing the test. If you change this test, it's a separate PR.